src/index.html

```diff
@@ -20,8 +21,331 @@
+        >
+          BUY
+        </a>
+      </div>
+      <!-- #endregion -->
+      <!-- #region card -->
+      <div class="card">
+        <img
+          class="card__image"
+          src="/src/images/imac.jpeg"
+          alt="Product"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__subtitle">Product code: 195434</p>
+
+        <div class="card__response">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__text">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <p class="left">Price:</p>
+          <p class="right">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="card__buy"
+        >
+          BUY
+        </a>
+      </div>
+      <!-- #endregion -->
+      <!-- #region card -->
+      <div class="card">
+        <img
+          class="card__image"
+          src="/src/images/imac.jpeg"
+          alt="Product"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__subtitle">Product code: 195434</p>
+
+        <div class="card__response">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__text">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <p class="left">Price:</p>
+          <p class="right">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="card__buy"
+        >
+          BUY
+        </a>
+      </div>
+      <!-- #endregion -->
+      <!-- #region card -->
+      <div class="card">
+        <img
+          class="card__image"
+          src="/src/images/imac.jpeg"
+          alt="Product"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__subtitle">Product code: 195434</p>
+
+        <div class="card__response">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__text">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <p class="left">Price:</p>
+          <p class="right">$2,199</p>
+        </div>
+        <a
+          href="#"
```

Comment:
Using `href="#"` is not recommended as it can cause unexpected behavior. Consider replacing it with a valid URL or handling the click event with JavaScript.